test-dev-dssspell.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
I think `test-dev-dssspell.sh` can be removed, following the changes introduced in fdc1a0b